tests/unit/test_clidriver.py

```diff
@@ -773,6 +781,9 @@ def test_lineage(self):
         self.cmd.lineage = ['foo']
         self.assertEqual(self.cmd.lineage, ['foo'])
 
+    def test_lineage_names(self):
```

Comment:
This isn't technically wrong, but it's a little confusing given the test above has `self.assertEqual(self.cmd.lineage, ['foo'])`.  Do you think it's worth changing the `test_lineage` test to set the lineage to an object to be consistent with real world usage?